perf-test/app/page.tsx

```diff
@@ -459,6 +481,16 @@ export default function Home() {
                   ></Player>
                 )
               }
+              {
+                playerId == 4 && (
+                  <SkottiePlayer
+                    lottieURL={anim.lottieURL}
+                    mode={SkottieRenderMode.CPU}
```

Comment:
Could you briefly compare the perf(memory/fps/lighthouse) between CPU/GPU?  We could choose a better one.

Reply:
The SkottiePlayer component supports GPU canvas. However it only allows 16 animations in my machine, which is maximum hardware concurrency. Still I don't know the reason why, I followed [skia's example code](https://github.com/google/skia/blob/main/tools/skottie-wasm-perf/skottie-wasm-perf.html#L27). Need to figure out to expand the limitation.

So I removed Skia GPU option in this PR.

Reply:
\+ @kudanai